examples/demo/Standard_Editors/ImageEnumEditor_demo.py

```diff
@@ -24,40 +32,45 @@ class Dummy(HasTraits):
     """
     x = Str()
 
-    view = View()
+    traits_view = View()
 
 
 class ImageEnumEditorDemo(HasTraits):
     """ Defines the ImageEnumEditor demo class.
     """
 
     # Define a trait to view:
-    image_from_list = Trait(editor=ImageEnumEditor(values=image_list,
-                                                   prefix='@icons:',
-                                                   suffix='_origin',
-                                                   cols=4,
-                                                   klass=Dummy),
-                            *image_list)
+    image_from_list = Enum(
+        *image_list,
+        editor=ImageEnumEditor(
+            values=image_list,
+            prefix='@icons:',
+            suffix='_origin',
+            cols=4,
+            klass=Dummy
+        )
+    )
 
     # Items are used to define the demo display, one Item per editor style:
     img_group = Group(
         Item('image_from_list', style='simple', label='Simple'),
         Item('_'),
-        Item('image_from_list', style='custom', label='Custom'),
-        Item('_'),
         Item('image_from_list', style='text', label='Text'),
         Item('_'),
-        Item('image_from_list', style='readonly', label='ReadOnly')
+        Item('image_from_list', style='readonly', label='ReadOnly'),
+        Item('_'),
+        Item('image_from_list', style='custom', label='Custom')
```

Comment:
With wx toolkit, the popup from simple style editor would blend with the custom editor and it looked like the simple editor was broken. To avoid this I moved the custom style to the bottom of the window.